testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/chat/ChatWindow.kt

```diff
@@ -81,33 +78,34 @@ class ChatWindow(private val activity: Activity) {
     ) {
         if (hasContact) {
             this.specificContactId = contactId
-            this.message = message
-            this.onSuccessSpecificContactId = onSuccess
-            this.onErrorContact = onError
+            if (getSavedSpecificContact() != null) {
+                this.message = message
+                this.onSuccessSpecificContactId = onSuccess
+                this.onErrorContact = onError
 
-            prepareWindow(ContactSelectionMode.OTHER)
-            prepareDataForAdapter(ContactSelectionMode.OTHER)
+                prepareWindow(ContactSelectionMode.OTHER)
+                prepareDataForAdapter(ContactSelectionMode.OTHER)
 
-            // preview dialog
-            contactSelectionAlertDialog.show()
+                contactSelectionAlertDialog.show()
+            } else showInstruction("Provided contact id hasn't been saved in HostApp yet.")
         } else warnNoContactSaved()
     }
 
     private fun prepareWindow(mode: ContactSelectionMode) {
         val rootView = WindowChatBinding.inflate(layoutInflater, null, false)
+        contactSelectionAlertDialog =
+            AlertDialog.Builder(activity, R.style.AppTheme_DefaultWindow).create()
+        contactSelectionAlertDialog.setView(rootView.root)
 
-        // set message content
-        GlobalScope.launch(Dispatchers.Main) {
-            message.apply {
-                rootView.messageImage.load(activity, image, R.drawable.r_logo)
```

Comment:
Looks like, Glide has some issues with thread while loading image from different kind of urls. Removing this line temporarily fix the crash (MINI-3099), but currently it's showing the default icon. I am still working this.